torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -779,6 +787,90 @@ def perspective(
         return perspective_image_tensor(inpt, perspective_coeffs, interpolation=interpolation, fill=fill)
 
 
+def elastic_image_tensor(
+    img: torch.Tensor,
+    displacement: torch.Tensor,
+    interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+    fill: Optional[List[float]] = None,
+) -> torch.Tensor:
+    return _FT.elastic_transform(img, displacement, interpolation=interpolation.value, fill=fill)
+
+
+def elastic_image_pil(
+    img: PIL.Image.Image,
+    displacement: torch.Tensor,
+    interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+    fill: Optional[Union[int, float, Sequence[int], Sequence[float]]] = None,
+) -> PIL.Image.Image:
+    t_img = pil_to_tensor(img)
+    fill = _convert_fill_arg(fill)
```

Comment:
It's a bit weird that `_convert_fill_arg()` calls exist within `*_pil` kernels but not for `_tensor` ones. I understand you do this because that's not JIT-scriptable for tensor and that we don't care about JIT-scriptability for PIL. What are the implications of phasing this out from all `*_pil` kernels and instead move them to the high-level kernels? This comment applies to all kernels.

Reply:
This is a particular case when we call `elastic_image_tensor` from `elastic_image_pil` as we do not implement natively elastic transformation on pillow. So, the sequence of 2 ops:
```
fill = _convert_fill_arg(fill)
output = elastic_image_tensor(t_img, displacement, interpolation=interpolation, fill=fill)
```
is the same as in a Image.elastic implementation: https://github.com/pytorch/vision/pull/6305/files#diff-f49325ccedde943495459399bf815a68db299bdd344f2e8309e87934a106b79cR246-R257

Reply:
Right, but why not move all these `_convert_fill_arg()` calls directly on the high-level kernels to avoid calling them on the low-level kernels over and over?

Reply:
OK, I can remove it from here and do that for this particular case. I can see that there should be no other similar implementation.

Reply:
As discussed offline, let's keep the actual code as we keep uniform method signatures